src/gui/DatabaseOpenWidget.cpp

```diff
@@ -171,8 +171,7 @@ bool DatabaseOpenWidget::event(QEvent* event)
     bool ret = DialogyWidget::event(event);
 
     switch (event->type()) {
-    case QEvent::Show:
-    case QEvent::WindowActivate: {
+    case QEvent::Show: {
```

Comment:
Window activate is important. The window can already be visible, but not in focus/active, e.g., after Alt-Tab, mouse focus, or after screen unlock.

Reply:
Then we should just trigger a polling on window activate and not a registration.

Reply:
What do you mean by registration? We could skip the polling on Show, but for consistency, I would prefer to poll everywhere. Overlapping polls would be cancelled anyway.

Reply:
You end up registering a new set of listeners every tike the window is activated before this change.

Reply:
Ah, that's what you mean. Yeah, I suppose that's left over from the first implementation when you could register only one trigger. Just skip the registration on WindowActivate, but still poll and, first of all, update the quick unlock state.